### ISSUE TITLE
minor path suggestion

### ISSUE THREAD

**Issue by michaelkarlcoleman** [OleHolmNielsen/Slurm_tools#23]
This is great--thanks!

It would be slightly easier to set up if it just used the 'awk' and SLURM commands already in $PATH, which is likely the usual case.

**Comment by clue2** [comment]
I agree - our environment was set up using Nvidia DeepOps (https://github.com/NVIDIA/deepops) which places slurm under /usr/local/bin/ while awk is still under /usr/bin/ so I've had to edit pestat before rolling it out

**Comment by OleHolmNielsen** [comment]
Which of these Slurm tools are you commenting on?
Can you suggest how to make the PATH more flexible in the script?
Thanks,
Ole

**Comment by michaelkarlcoleman** [comment]
Apologies for the delayed reply.  Looking just now, I think you've fixed this, or perhaps I was simply mistaken.  --Mike
